lib/nds_explore.rb

```diff
@@ -0,0 +1,18 @@
+require 'directors_database'
+
+def pretty_print_nds
+  nds = directors_database
+  pp nds
+  nil
+end
+
+def print_first_directores_movie_title
+  ss_movies = directors_database[0][:movies]
+  index = 0
+
+  while index < ss_movies.length do
+    titles = ss_movies[index][:title]
+    pp titles
```

Comment:
Here I think `puts` is the right method. We want to teach `puts || pp`. That you can mix them, as this demonstrates, is cool, but `pp big_old_structure` or iterate `big_old_structure` is the goal.